### PR TITLE
Fix Subm broker creation in 2.5 https://issues.redhat.com/browse/ACM-2029

### PR DESCRIPTION
**Pull request by oafischer**
https://issues.redhat.com/browse/ACM-2029

Also for 2.6 and 2.7.

### CODE REVIEW

File: add-ons/submariner/submariner.adoc [hunk 1]
```diff
@@ -119,7 +119,13 @@ The `ClusterAdmin` role has the required permissions to create this object in th
 
 Complete the following steps to create the `submariner-broker` object:
 
-. Create `submariner-broker` object that specifies the Globalnet configuration by creating a YAML file named `submariner-broker.yaml` that contains content that resembles the following example: 
+. Retrieve the `broker-namespace`by running the following command:
```
Comment (dockerymick) [inline add-ons/submariner/submariner.adoc]: space need between the code block tick mark and "by"

File: add-ons/submariner/submariner.adoc [hunk 2]
```diff
@@ -132,9 +138,11 @@ spec:
   globalnetEnabled: <true-or-false>
 ----
 +
-Replace `broker-namespace` with the name of your broker namespace.
+Replace `broker-namespace` with the name of your broker namespace. 
 +
 Replace `true-or-false` with `true` to enable Globalnet. 
++
+*Note:* The `metadata``name` parameter must be `submariner-broker`.
```
Comment (dockerymick) [inline add-ons/submariner/submariner.adoc]: I usually create a parameter breadcrumb. I suggest changing `metadata``name` to `metadata.name` OR add a space between metadata and name (`metadata` `name`)

**Review by dockerymick — APPROVED** [review]
Made some comments, otherwise looks good!

File: add-ons/submariner/submariner.adoc (hunk 1 above)
Comment (manosnoam) [reply add-ons/submariner/submariner.adoc]: And I think you can say: Retrieve the \<broker-namespace\>

**Review by manosnoam — COMMENTED** [review]
lgtm

**Pull request merged**